.bit/responses/5.5-Week 5 Step 5.md

```diff
@@ -0,0 +1,29 @@
+---
+files: n/a
+stepType: comment
+scripts: n/a
+week: 5
+step: 5
+name: Week 5 Step 5
+---
+Week 5 Step 5 ◯◯◯◯⬤◯ | 🕐 Estimated completion: 1-5 hours
+
+## Start Coding!
+
+### ✅  Task:
+
+- [ ] Based on your schedule, begin your first task
+- [ ] Commit your code in a new branch named after the component
+- [ ] Comment a description of which component you completed to move on
+
+### Developing the first component
+
+Let's look back at the advice we got from the previous step: "start small then build big." Your goal is to get something working - not to developing your final project in 1 week.
+* Begin by just having one working component of the project. Ex:
+  * Press a button on your webpage and receive information
+  * Create a working Azure Function that can return information
+* After that's complete and tested, begin building onto that with more and more features!
+
+### Committing code
+
```

Comment:
Also add here that they should **name** commits correctly so that they are easy to search for